ci/check-documentation.py

```diff
@@ -114,6 +116,21 @@ def complain(message):
             if md_path in index_child_md_links:
                 referenced_md_files.add(target_path)
 
+        # check <omz_dir> reference validity
+
+        for link in doc_page.code_spans():
+            if link.startswith(omz_prefix):
+                file_relative_path = link[len(omz_prefix):]
+                file_path = OMZ_ROOT / file_relative_path
+
+                if ".." in file_relative_path:
```

Comment:
This doesn't check for `..` components - it checks whether the string contains the two characters `..` anywhere. You should turn `file_relative_path` into a `Path` and use `file_relative_path.parts` here.